src/map/mob.h

```diff
@@ -202,6 +202,7 @@ struct mob_data {
 	int target_id,attacked_id;
 	int areanpc_id; //Required in OnTouchNPC (to avoid multiple area touchs)
 	unsigned int bg_id; // BattleGround System
+	unsigned int clan_id; // Clan System
```

Comment:
Clan id is signed elsewhere, so it should be here as well. Please remove the `unsigned`